countingLoops/script.js

```diff
@@ -0,0 +1,35 @@
+for (let i = 1; i <= 10; i++) {
+    console.log (i);
+  } for (let i = 10; i >= 1; i--)  {
+        console.log(i); 
+}
```

Comment:
These are two separate loops

```suggestion
for (let i = 1; i <= 10; i++) {
    console.log (i);
}

for (let i = 10; i >= 1; i--)  {
    console.log(i);
}
```